Remove use of deprecated FrameGrabberInterfaces.h YARP header

traversaro opened the pull request:
The header was deprecated since before YARP 3.9 (see https://github.com/robotology/gazebo-yarp-plugins/pull/698), and it was removed in YARP 3.12, so removing its use fix the compilation against YARP 3.12 .

GiulioRomualdi reviewed (approved):
Thank you @traversaro

traversaro commented:
> I think the ci errors are not related to this PR, right?

Yes, that is correct. The CI failures are due to:
* Ubuntu 20.04 runners being dropped by GitHub actions, fixed by https://github.com/ami-iit/bipedal-locomotion-framework/pull/972
* Latest opencv package on conda-forge migrated to VS2022, fixed by https://github.com/ami-iit/bipedal-locomotion-framework/pull/974
